scripts/vim-patch.sh

```diff
@@ -892,21 +892,20 @@ is_na_patch() {
   local NA_REGEXP="$NVIM_SOURCE_DIR/scripts/vim_na_regexp.txt"
   local NA_FILELIST="$NVIM_SOURCE_DIR/scripts/vim_na_files.txt"
 
-  local FILE_LNUM
-  # shellcheck disable=SC2086
-  FILE_LNUM="$(diff <(git -C "${VIM_SOURCE_DIR}" diff-tree --no-commit-id --name-only -r "$patch" | grep -v -f "$NA_REGEXP") "$NA_FILELIST" |
-    grep -c '^<')" || FILE_LNUM=0
-  test "$FILE_LNUM" -gt 1 && return 1
-  if test "$FILE_LNUM" -eq 1; then
+  local FILES_REMAINING
+  FILES_REMAINING="$(diff <(git -C "${VIM_SOURCE_DIR}" diff-tree --no-commit-id --name-only -r "$patch" | grep -v -f "$NA_REGEXP") "$NA_FILELIST" |
+    grep '^<')"
```

Comment:
`||` is to handle empty string for this `grep` or broken pipe due to `diff` getting an empty file. Had issues with is_na_patch() aborting early without `||`.

Reply:
I can't seem to reproduce the problem, but I'll add a `|| true`.